.github/workflows/dockerhub.yml

```diff
@@ -32,11 +32,15 @@ on:
         type: "string"
         required: false
       tag_name_suffix:
-        description: "Optional: Custom docker image tag name suffix to be appended, defaults to empty string"
+        description: "Optional: Custom docker image tag name suffix to be appended (applies to both base image and output image unless remove_suffix is used), defaults to empty string"
         type: "string"
         required: false
       replace_suffix:
-        description: "Optional: True if last part of the docker_tag needs to be replaced with given tag_name_suffix"
+        description: "Optional: True if last part of the docker_tag needs to be replaced with given tag_name_suffix (applies to both base image and output image unless remove_suffix is used)"
+        type: "boolean"
+        required: false
+      remove_suffix:
```

Comment:
If both replace_suffix and remove_suffix are set to true, do we run both transformations? In which order? may be good to define the intended behavior or at least gives a warning if both are simultaneously true ?

Reply:
Intended design is to only switch the last dash, so both should be modifying the last suffix. In that case, they would cancel out becauser order is first replace suffix but then remove the suffix later anyways. So it is benign to have both enabled, or a.k.a removal procedure will take effect. I added print to notify

Reply:
If it doesn't have any dashes it keeps the original because theoretically there was no "suffix"